src/test/resources/test-gitlab4j-sample.properties

```diff
@@ -0,0 +1,4 @@
+TEST_NAMESPACE=some-namespace
+TEST_PROJECT_NAME=some-project-name
+TEST_HOST_URL=some-gitlab-url
+TEST_PRIVATE_TOKEN=some-private-token
```

Comment:
Goes along with the above comment. Gives a new contributor an idea of how to setup for testing.